### PR TITLE
Base architecture for React app in Typescript.

### PR DESCRIPTION
**Pull request by XshubhamX**
**What kind of change does this PR introduce?**

Enhancement

**Summary**

I have created the base architecture for the application. Next step is to update README.md file.

### CODE REVIEW

**Comment by XshubhamX** [comment]
@xsh2047 and @DangaRanga sir please review.

**Review by XshubhamX — COMMENTED** [review]
@xsh2047 and @DangaRanga sir please review.

**Comment by yasharth291** [comment]
Hey @XshubhamX  i think yarn.lock is a log file and should be added in the gitignore along with package.lock

File: src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,8 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => {
```
Comment (DangaRanga) [inline src/App.tsx:4]: I would recommend sticking to explicitly defining your functional components with the `function` keyword as opposed to defining them with arrow functions, to improve the readability in the long run, similar to how they are defined in the React docs.

https://reactjs.org/docs/components-and-props.html

**Comment by palisadoes** [comment]
@XshubhamX What issue does this resolve? We are only approving PRs tied to a pre-existing issue.

**Review by DangaRanga — COMMENTED** [review]
> @XshubhamX What issue does this resolve? We are only approving PRs tied to a pre-existing issue.

I believe this PR is referencing #23

File: src/App.tsx (hunk 1 above)
Comment (XshubhamX) [reply src/App.tsx:4]: ok sir i will change @DangaRanga sir

**Review by DangaRanga — APPROVED** [review]

**Review by XshubhamX — COMMENTED** [review]
Resolved conflicts

**Comment by yasharth291** [comment]
@XshubhamX please add yarn.lock in the gitignore

**Review by yasharth291 — CHANGES_REQUESTED** [review]
@XshubhamX please add yarn.lock in the gitignore

**Pull request merged**